src/particles/diagnostics/DiagnosticOutput.cpp

```diff
@@ -70,6 +71,35 @@ namespace impactx::diagnostics
                                 << px << " " << py << " " << pt << "\n";
                     } // i=0...np
                 } // if( otype == OutputType::PrintParticles)
```

Comment:
Above are the lines `amrex::AllPrintToFile(file_name) << "x y t px py pt\n";` which print the table header.

We need to update those as well depending on `otype`.